ftp-ballerina/commons.bal

```diff
@@ -17,40 +17,38 @@
 import ballerina/io;
 
 # Represents the set of protocols supported by the FTP listener and client
-public type Protocol "ftp"|"sftp"|"ftps";
+public type Protocol "ftp"|"sftp";
 
 # Underlying communication happens using FTP
 public const FTP = "ftp";
 # Underlying communication happens using SFTP
 public const SFTP = "sftp";
-# Underlying communication happens using FTPS.
-public const FTPS = "ftps";
 
 # Configuration to connect to a trust store.
 #
 # + path - Path to a trust store file
 # + password - Trust store password
 public type TrustStore record {|
-    string? path = ();
-    string? password = ();
+    string path?;
+    string password?;
 |};
 
 # Configuration to connect to a key store.
 #
 # + path - Path to the key store file
 # + password - Key store password
 public type KeyStore record {|
```

Comment:
Shall we use `crypto:KeyStore` instead of this one? The other stdlibs follow that pattern.

Reply:
Removed `KeyStore` till FTPS is implemented.